scapy/fields.py

```diff
@@ -428,16 +428,20 @@ def getfield(self, pkt, s):
 
 
 class PacketListField(PacketField):
-    __slots__ = ["count_from", "length_from"]
+    __slots__ = ["count_from", "length_from", "next_cls_cb"]
     islist = 1
-    def __init__(self, name, default, cls, count_from=None, length_from=None):
+    def __init__(self, name, default, cls=None, count_from=None, length_from=None, next_cls_cb=None):
         if default is None:
             default = []  # Create a new list for each instance
+        if not isinstance(next_cls_cb, type(None)):
```

Comment:
Thank you for your review. 

The isinstance() test is a provision in case someone would be subclassing None for unknown and weird reasons, but I can definetely live with `is not None`.

I also was assigning a valid Packet to class, just in case someone would subclass PacketListField and would expect `cls` to always be a valid Packet subclass.

Overall, I guess that sometimes I am a little too careful for Python :)